database.ts

```diff
@@ -0,0 +1,50 @@
+const Data = [
+  {
+    name: "Furious Express",
+    status: "moving locomotive",
+    route: "MG-ES",
+    load: "iron ore",
+    driver_id: "Amanda",
+    maneuverer: "Ana",
+  },
+  {
+    name: "Thunderbolt Runner",
+    status: "locomotive under maintenance",
+    route: "ES-MG",
+    load: "soybean",
+    driver_id: "André",
+    maneuverer: "Antonio",
+  },
+  {
+    name: "Ironclad Voyager",
+    status: "moving locomotive",
+    route: "ES-MG",
+    load: "soybean",
+    driver_id: "Caio",
+    maneuverer: "Carla",
+  },
+  {
+    name: "Lightning Express",
+    status: "stopped locomotive",
+    route: "MG-ES",
+    load: "iron ore",
+    driver_id: "Daniela",
+    maneuverer: "Douglas",
+  },
+  {
+    name: "Mighty Hauler",
+    status: "moving locomotive",
+    route: "ES-MG",
+    load: "soybean",
+    driver_id: "Juliana",
+    maneuverer: "Julio",
+  },
+  {
+    name: "Silver Arrow",
+    status: "stopped locomotive",
+    route: "MG-ES",
+    load: "iron ore",
+    driver_id: "Luiz",
+    maneuverer: "Laura",
+  },
+];
```

Comment:
❓Qual a importância desse database no código?

Reply:
a principio achei que iriamos puxar que nem fizemos na fakeStore, seria a nossa seed, mas creio q devemos fazer tudo no supabase mesmo. Se for so por la, ai podemos retirar esse database

Reply:
Se os dados que formos colocar no supabase forem esses, podemos manter e adicioná-los no banco depois com código mesmo, é mais fácil do que adicionar um por um no supabase.